packages/shared/src/cookie.ts

```diff
@@ -13,8 +13,8 @@ export function createCookieHandler(cookieName: string) {
     /**
      * Setting a cookie will use some defaults such as path being set to "/".
      */
-    set(newValue: string, options: Cookies.CookieAttributes = {}) {
-      return Cookies.set(cookieName, newValue, options);
+    set(newValue: string, options: Cookies.CookieAttributes = {}): void {
+      Cookies.set(cookieName, newValue, options);
```

Comment:
This could be a breaking change but since it's in the `@clerk/shared` package and it's supposed to be used internally i think we are good to go.

Reply:
❓ why did we remove the `return` statement here?

Reply:
https://github.com/clerk/javascript/pull/3362#issuecomment-2110070935